mappings/net/minecraft/unmapped/C_unqwyjvr.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/unmapped/C_unqwyjvr
+	CLASS C_sdkmrqke
+		METHOD m_kueojymf sample (III)D
+	CLASS C_utnexdzf NoiseInterpolator
+		FIELD f_mejpfspb startNoiseBuffer [[D
+		FIELD f_sfmejueb endNoiseBuffer [[D
+		METHOD m_cohgrnoy swapBuffers ()V
+		METHOD m_gdohomuy sampleStartNoise ()V
+		METHOD m_wjebstow sampleNoise ([[DI)V
+			ARG 1 buffer
+			ARG 2 noiseX
+		METHOD m_wpfrryvm sampleEndNoise (I)V
+			ARG 1 x
+	CLASS C_ydritevk
+		METHOD <init> (Lnet/minecraft/unmapped/C_wgwzecpu;II)V
+			ARG 2 x
+			ARG 3 z
```

Comment:
The rest of the `NoiseInterpolator` mappings (fields and methods) could be added back to this class.